modules/apps/headless/headless-form/headless-form-impl/src/main/java/com/liferay/headless/form/internal/resource/v1_0/FormResourceImpl.java

```diff
@@ -60,6 +65,17 @@ public Form getForm(Long formId) throws Exception {
 				contextCompany.getCompanyId(), siteId));
 	}
 
+	@Override
+	public FormDocument postFormUploadFile(
+			@NotNull Long formId, MultipartBody multipartBody)
+		throws Exception {
+
+		return _uploadFileHelper.toDocument(
+			_uploadFileHelper.uploadFile(
```

Comment:
If you are not using uploadFile anywhere, define a method here, instead of using the helper. Brian doesn't like helpers.